pkg/storage/store.go

```diff
@@ -119,34 +119,40 @@ func StartTask(id int) {
 		panic("Task " + string(id) + " does not exist.")
 	}
 
-	startTime := time.Now()
+	// Refactor: add flag so these if's can be cleaner
+	if isTimeUnset(task.EndTime) {
```

Comment:
Can you flip the check here? Since you add the check this way, the meat of the function is indented with one little line at the bottom. If you reversed it, it would read better.

```
if !isTimeUnset(task.EndTime) {
    handleError(fmt.Errorf("..."))
    return
}

// Do interesting stuff
```